network-api/networkapi/wagtailpages/tests/test_blog.py

```diff
@@ -835,10 +835,8 @@ def test_authors_detail(self):
         response = self.client.get(path=blog_author_url)
         self.assertTemplateUsed(response, "wagtailpages/blog_author_detail_page.html")
         self.assertContains(response, self.profile_1.name)
-        self.assertContains(response, self.profile_1.tagline)
```

Comment:
It was requested to not include Profile taglines in the author detail page, so I removed these from the tests!